hooks/useUserIdentity.ts

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react'
+import AccountsSDK from '@livechat/accounts-sdk'
+import { Config } from 'lib/config'
+
+type UserIdentity = {
+  account_id: string
+  access_token: string
+  expires_in: number
+  organization_id: string
+  scope: string
+  token_type: 'Bearer'
+}
+
+const accountsSDK = new AccountsSDK({
+  client_id: Config.lcClientId,
+  server_url: Config.lcAccountsURL,
+})
+
+function useUserIdentity() {
+  const [userIdentity, setUserIdentity] = useState<UserIdentity | null>(null)
+  const [loading, setLoading] = useState<boolean>(false)
+
+  const authorize = () => {
+    setLoading(true)
+    accountsSDK
+      .popup()
+      .authorize()
+      .then(setUserIdentity)
+      .finally(() => setLoading(false))
+  }
+
+  const logout = () => {
```

Comment:
Please make it async so users could wait for the logout function ends and then run some other logic